src/nni_manager/core/nnimanager.ts

```diff
@@ -281,6 +281,26 @@ class NNIManager implements Manager {
         return this.dataStore.getMetricData(trialJobId, metricType);
     }
 
+    public async getLatestMetricData(): Promise<MetricDataRecord[]> {
+        // FIXME: this can take a long time
```

Comment:
Is it better to implement this in sqlDatabase.ts ?

Reply:
Yes of course, but maybe in next release...